pyrsa/rdm/calc.py

```diff
@@ -48,7 +58,8 @@ def calc_rdm(dataset, method='euclidean', descriptor=None, noise=None):
         elif method == 'mahalanobis':
             rdm = calc_rdm_mahalanobis(dataset, descriptor, noise)
         elif method == 'crossnobis':
-            rdm = calc_rdm_crossnobis(dataset, descriptor, noise)
+            rdm = calc_rdm_crossnobis(dataset, descriptor, noise,
+                                      cv_descriptor=cv_descriptor)
```

Comment:
`calc_rdm_crossnobis(dataset, descriptor, noise, cv_descriptor) `
Would be sufficient